pkgs/tools/security/gallia/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "gallia";
+  version = "1.0.3";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "Fraunhofer-AISEC";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
r-ryantm would modify it with the first update.